### PR TITLE
add GetParameter up/down command

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -158,6 +159,51 @@ export interface IEventMtxStatus {
...
+export interface IParameterAbsoluteDataStatusMC extends IParameterAbsoluteDataStatus {
+    /**
+     * Channel that accept status changing.
+     */
+    channel: number
+}
+
+export interface IParameter {
+    id: number,
+    data: IParameterInitialData |
```
Comment (DarkPark) [inline src/CommandBinaryBuffer.ts:201]:
```suggestion
    data: TParameterData
```
?

File: src/commands/uplink/GetParameter.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            data: {value: 2023, meterValue: 204, pulseCoefficient: 100}
+        },
+        hex: {
+            header: '04 0a',
+            body: '17 00 00 00 cc 82 00 00 07 e7'
+        }
+    },
+    {
+        name: 'device sending absolute data',
+        parameters: {id: deviceParameters.ABSOLUTE_DATA_STATUS, data: {status: 1}},
```
Comment (DarkPark) [inline src/commands/uplink/GetParameter.ts:24]: let's split it to 4 lines

File: src/commands/downlink/GetParameter.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import {DOWNLINK} from '../../constants/directionTypes.js';
+import * as deviceParameters from '../../constants/deviceParameters.js';
+
+
+/**
+ *GetParameter command parameters
```
Comment (DarkPark) [inline src/commands/downlink/GetParameter.ts:8]:
```suggestion
 * GetParameter command parameters
```

File: src/commands/downlink/GetParameter.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+ *
+ * @example
+ * import {constants} from 'jooby-codec';
+ * {id: constants.deviceParameters.INITIAL_DATA, data: {value: 2023, meterValue: 204, pulseCoefficient: 100}}
```
Comment (DarkPark) [inline src/commands/downlink/GetParameter.ts:12]: interface has only id

File: src/commands/downlink/GetParameter.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    {
+        name: 'request initial data from device',
+        parameters: {id: deviceParameters.INITIAL_DATA},
+        hex: {
+            header: '04 01',
+            body: '17'
+        }
+    },
+    {
+        name: 'request status of data type sending, absolute or another',
```
Comment (DarkPark) [inline src/commands/downlink/GetParameter.ts:36]: let's rephrase

File: src/CommandBinaryBuffer.ts [hunk 6]
```diff
@@ -158,6 +159,51 @@ export interface IEventMtxStatus {
...
+    IParameterInitialDataMC |
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts:202]: reworked

File: src/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts:201]: added

File: src/commands/uplink/GetParameter.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/commands/uplink/GetParameter.ts:24]: done

File: src/commands/downlink/GetParameter.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/commands/downlink/GetParameter.ts:8]: fixed

File: src/commands/downlink/GetParameter.ts (hunk 4 above)
Comment (DeadByElpy) [reply src/commands/downlink/GetParameter.ts:12]: fixed

File: src/commands/downlink/GetParameter.ts (hunk 5 above)
Comment (DeadByElpy) [reply src/commands/downlink/GetParameter.ts]: done

File: src/commands/uplink/GetParameter.ts (hunk 2 above)
Comment (DarkPark) [reply src/commands/uplink/GetParameter.ts]: this is ok
```ts
parameters: {
    id: deviceParameters.ABSOLUTE_DATA_STATUS,
    data: {status: 1}
},
```

File: src/commands/downlink/GetParameter.ts [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+import {DOWNLINK} from '../../constants/directionTypes.js';
+import * as deviceParameters from '../../constants/deviceParameters.js';
+
+
+/**
+ * GetParameter command parameters
+ *
+ * @example
+ * import {constants} from 'jooby-codec';
+ * {id: constants.deviceParameters.INITIAL_DATA}
```
Comment (DarkPark) [inline src/commands/downlink/GetParameter.ts:12]:
```suggestion
 *
 * const parameters = {id: constants.deviceParameters.INITIAL_DATA};
```

File: src/commands/uplink/GetParameter.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/commands/uplink/GetParameter.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**